xml/http_servers.xml

```diff
@@ -1689,13 +1689,21 @@
     <param pos="0" name="service.component.product" value="JBossWeb"/>
     <param pos="2" name="service.component.version"/>
   </fingerprint>
-  <fingerprint pattern="^Servlet/[\d\.]+; JBossAS-(.*)$">
+  <fingerprint pattern="^Servlet\/[\d\.]+; JBossAS-(.*)$">
```

Comment:
The regex doesn't work in Rubular, just to be safe.

http://rubular.com/r/Yeh1PVY2RU